components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -101,7 +101,7 @@ export class TeamDBImpl extends TransactionalDBImpl<TeamDB> implements TeamDB {
             return {
                 userId: u.id,
                 fullName: u.fullName || u.name,
-                primaryEmail: User.getPrimaryEmail(u),
+                primaryEmail: getPrimaryEmail(u),
```

Comment:
Would it be alright if DB layer only returns pair [user, memebership], but conversion happens in org service in server?

Reply:
will do